FHICORC/Locales/nb.json

```diff
@@ -11,16 +11,16 @@
   "SETTINGS_FETCH_CONTRL_KEYS": "Hent kontrolldata",
   "SETTINGS_LAST_UPDATED": "Sist oppdatert: {0}",
   "SCANNER_CAMERA_PERMISSION_TITLE": "Koronasertifikat vil ha tilgang til kamera",
-  "SCANNER_CAMERA_PERMISSION_DESC": "Tillat tilgang til kamera for å komme i gang med å scanne sertifikater.",
-  "SCAN_CORONAPAS_DESCRIPTION": "Scan koronasertifikat for å se covid-19-tilstand.",
-  "SCAN_CORONAPAS": "Scanner",
-  "ZEBRA_SCANNER_TITLE": "Scan koronasertifikat",
+  "SCANNER_CAMERA_PERMISSION_DESC": "Tillat tilgang til kamera for å komme i gang med å skanne sertifikater.",
+  "SCAN_CORONAPAS_DESCRIPTION": "Skan koronasertifikat for å se covid-19-tilstand.",
+  "SCAN_CORONAPAS": "Skanner",
+  "ZEBRA_SCANNER_TITLE": "Skan koronasertifikat",
```

Comment:
I think "Skan" should be "Skann" here and on line 15. From dictionary: [uib](https://ordbok.uib.no/perl/ordbok.cgi?OPP=SKANn&ant_bokmaal=5&ant_nynorsk=5&begge=+&ordbok=begge ) og [naob](https://naob.no/ordbok/skann)

Reply:
Yes, I have updated it now